src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url('../images/star-active.svg');
+    }
+  }
```

Comment:
The SCSS `@for` directive is used to generate modifiers for the `stars` block. According to the Code Style Rules, ensure that the indentation for the contents of the `@for` directive is consistent with the rest of the file. Typically, this means you should indent the contents inside the `@for` block with two spaces.